qiskit/extensions/quantum_initializer/isometry.py

```diff
@@ -64,7 +64,7 @@ class Isometry(Instruction):
     # finally, we convert the labels back to the qubit numbering used in Qiskit
     # (using: _get_qubits_by_label)
 
-    def __init__(self, isometry, num_ancillas_zero, num_ancillas_dirty):
+    def __init__(self, isometry, num_ancillas_zero, num_ancillas_dirty, eps=1e-10):
```

Comment:
```suggestion
    def __init__(self, isometry, num_ancillas_zero, num_ancillas_dirty, eps=_EPS):
```
for consistency with a bunch of other files which hardcode the precision at the top, yet allow user setting. e.g. see: `qiskit/quantum_info/synthesis/one_qubit_decompose.py`